src/braket/experimental/autoqasm/operators/assignments.py

```diff
@@ -15,7 +15,7 @@
 """Operators for assignment statements."""
 
 import copy
-from typing import Any
+from typing import Any, Iterable
```

Comment:
[nit] could use `collections.abc.Iterable`, technically` typing.Iterable` is deprecated.

Reply:
good catch